public/template/layout.jade

```diff
@@ -8,15 +8,15 @@ html(lang="en")
 
     if config.noCDN
 
-      link(href="/stylesheet/bootstrap.min.css", rel="stylesheet")
-      link(href="/stylesheet/bootstrap-responsive.min.css", rel="stylesheet")
+      link(href="/stylesheet/bootstrap-combined.no-icons.min.css", rel="stylesheet")
```

Comment:
Sorry, I meant, what is it? Is it a concatenation of the regular Bootstrap stylesheets and the `-responsive` stylesheets?